src/utils.js

```diff
@@ -23,3 +23,29 @@ export const getCommentsByArticleId = (id) => {
       return res.data;
     });
 };
+
+export const increaseVoteById = (id, vote) => {
+  return axios
+    .patch(`https://news-api-5rlq.onrender.com/api/articles/${id}`, {
+      inc_votes: +1,
+    })
+    .then((res) => {
+      return res.data.vote;
+    })
+    .catch((err) => {
+      return "failed to fetch counts";
+    });
```

Comment:
You need to handle your errors at the end of your chain, not in here. 

This message isnt great as well for a user on the website. Do they know what fetch counts means?